pkgs/os-specific/linux/dpdk/default.nix

```diff
@@ -71,5 +75,6 @@ in stdenv.mkDerivation rec {
     license = with licenses; [ lgpl21 gpl2 bsd2 ];
     platforms =  platforms.linux;
     maintainers = with maintainers; [ magenbluten orivej ];
+    broken = stdenv.isAarch64; # only supports armv8
```

Comment:
I don't believe this makes sense like this?

Reply:
It's from https://github.com/NixOS/nixpkgs/pull/122588/checks?check_run_id=2560790845

Reply:
To clarify: aarch64 is armv8, so I don't think the comment is correct.

Reply:
oh, i see what you mean. Yea the comment will probably need to be updated that  it just doesn't build